src/bp/ui-admin/src/Pages/Workspace/Bots/index.tsx

```diff
@@ -73,7 +73,10 @@ class Bots extends Component<Props> {
   componentDidMount() {
     this.props.fetchBots()
     this.props.fetchBotHealth()
-    this.props.profile && this.props.profile.isSuperAdmin && this.props.fetchModules()
+
+    if (!this.props.modules.length && this.props.profile && this.props.profile.isSuperAdmin) {
```

Comment:
```suggestion
    if (!this.props.modules.length && this.props.profile && this.props.profile?.isSuperAdmin) {
```

Reply:
I don't know why but this does not build in admin and the build does not say why